### ISSUE TITLE
Network Drive File Path Support

### ISSUE THREAD

**Issue by hero887** [MrPig91/PSChiaPlotter#51]
Could you add support for a network attached storage for a final directory for farming plots? I assigned a letter to my NAS drive, but it doesn't populate as a selectable option in the PSChiaPlotter.

Currently Swar doesn't work with NAS either. So the Windows-using Chia community is in a real pickle here. Whadda you think?

**Comment by igorwhite** [comment]
Hello!
Swar works for me over the network. PSChiaPlotter haven't tried it yet. But I think it's a promising program.

**Comment by gomadosnitro** [comment]
how can i add  \\x.x.x.x\directory in the final directory?

**Comment by igorwhite** [comment]
I connect as a network drive

**Comment by gomadosnitro** [comment]
How?
I have mapped a network drive with a letter but it doesn't populate as a selectable option.

**Comment by igorwhite** [comment]
I wrote about Swar. PSChiaPlotter haven't tried it yet.

**Comment by gomadosnitro** [comment]
ah ok

**Comment by CDprotector** [comment]
+1 for Network drives as final destination, its missing from drop down menu when creating new job.

**Comment by igorwhite** [comment]
Yes, PSChiaPlotter  does not work with network drives. Swar works. Make it work with network drives. I liked the program.

**Comment by sk00t3r** [comment]
I too have a need for mapped network drive support. I currently am using a self-made plot manager but have been following this build and the YouTube videos. I have not had time to test but I think the following link will either give you mapped drive support or put you on the right path to getting it to work.

https://raw.githubusercontent.com/sk00t3r/PSChiaPlotter/main/PSChiaPlotter/Private/Get-ChiaVolume.ps1

**Comment by admn-monkey** [comment]
+1 for network drive support. At this time I'm using a NAS for storage. I use PSChiaPlotter in the CLI and I love it. Still playing around getting the timing and resources fine tuned and PSChiaPlotter makes that so much easier to do. Thanks!

**Comment by sk00t3r** [comment]
I have found a working workaround. On my drive home I had an idea of creating a symbolic link and wondered if the Plotter would write to it. Sure enough it works. I am not endorsing this solution but if you want to use the Plotter with a mapped drive here is the work around.

Open CMD as Administrator and type the following:

mklink /D C:\LinkName \\NetworkLocation\LocationName

If you want to use PowerShell open it again running as Administrator and type the following:

New-Item -ItemType SymbolicLink -Path C:\LinkName -Target \\NetworkLocation\LocationName

Then inside the Plotter just point to the folder you just created. The path does not have to be C:\ it can be any local physical drive. That's it.

To remove the link you can right click on the folder and click Delete right within Explorer. This will not delete the contents. You can also use rmdir ONLY in CMD. Do NOT use rmdir inside PowerShell and do NOT us the DEL command.

If you absolutely have to use PowerShell to remove the link use the following command:

cmd /c rmdir .\Target 

I am not responsible if you lose any data. Please make sure you know what you are doing. Thank you.

**Comment by MrPig91** [comment]
Great discussion here! I have added the network drives to PSChiaPlotter and the updated version is now available. Sorry I didn't have this on the initial release, I had crazy slow copy speeds when with my network drives that it made more sense to connect those drives directly to my plotting computer and then move them to my harvester machines after they were filled. Anyways, I got about two dozen comments asking for this, so I decided to add it real quick. 

It grabs the network drives with "Get-CimInstance Win32_MappedLogicalDisk".
